charts/sn-platform/values.yaml

```diff
@@ -2353,4 +2364,9 @@ istio:
     # gateway selector if it's not `istio: ingressgateway`
     selector: {}
     tls:
+      # use existing secret
       certSecretName: "[CERTIFICATE SECRET NAME IN ISTIO ROOT NAMESPACE]"
+      # use cert-manager to generate secret, only take effect if certSecretName is empty,
+      # also need to config certs.istio_internal_issuer/certs.istio_public_issuer
+      cert_name: "[ISTIO CERT NAME]"
```

Comment:
If the customer needs to set a cert name anyway, why not just stick to the `certSecretName`?

Reply:
Maybe we don't need to unify Istio TLS configuration with others, how about the implementation in my first commit.

Reply:
Do you mean [this one](https://github.com/streamnative/charts/pull/690/commits/912f4d00bc9d3754f4eebeccc34636251f73b9d7#diff-f8fce96bac42e794714091b347ba9ba3152d535f10477277716a2172e2d8566bR6)? It looks mostly good to me.